guardian/ArchiveHunterDownloadManager
# Ag vaultdoor

## What does this change?
Adding VaultDoor integration

## How to test
Start it up, make sure that the vaultdoor address of the instance you're going to target is set in preferences, then log into VaultDoor and attempt a bulk download

## How can we measure success?
People can download from vaultdoor

## Have we considered potential risks?
Internal network only, using a proven security method
